babel.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
   presets: ['razzle/babel', 'stage-0'],
   plugins: [
     'transform-decorators-legacy',
+    ['transform-class-properties', { spec: true }],
```

Comment:
This is completely required?

Reply:
This is of support for static propTypes in ES 6 classes.